test/jdk/javax/swing/JButton/PressedButtonRightClickTest.java

```diff
@@ -74,22 +74,27 @@ private static void test() {
         testRobot.mouseMove((loc.x + 100), (loc.y + 100));
 
         // Press the left mouse button
+        System.out.println("press BUTTON1_DOWN_MASK");
         testRobot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
         myButton.setText("Left button pressed");
-        testRobot.delay(1000);
+        testRobot.delay(500);
 
         // Press the right mouse button
+        System.out.println("press BUTTON3_DOWN_MASK");
         testRobot.mousePress(InputEvent.BUTTON3_DOWN_MASK);
         myButton.setText("Left button pressed + Right button pressed");
-        testRobot.delay(1000);
+        testRobot.delay(500);
 
         // Release the right mouse button
+        System.out.println("release BUTTON3_DOWN_MASK");
         testRobot.mouseRelease(InputEvent.BUTTON3_DOWN_MASK);
         myButton.setText("Right button released");
-        testRobot.delay(1000);
+        testRobot.waitForIdle();
+        testRobot.delay(500);
 
         // Test whether the button is still pressed
         boolean pressed = myButton.getModel().isPressed();
+        System.out.println("release BUTTON1_DOWN_MASK");
```

Comment:
Is this a debugging left over ?

Reply:
Yes, I think it might be helpful to keep it.